packaging/pkcs11-provider.spec

```diff
@@ -1,17 +1,28 @@
+#Enable gpg signature verification
+%bcond_with gpgcheck
+
 Name:          pkcs11-provider
 Version:       0.2
 Release:       1%{?dist}
 Summary:       A PKCS#11 provider for OpenSSL 3.0+
 License:       Apache-2.0
 URL:           https://github.com/latchset/pkcs11-provider
 Source0:       %{url}/releases/download/v%{version}/%{name}-%{version}.tar.xz
+%if %{with gpgcheck}
+Source1:       %{url}/releases/download/v%{version}/%{name}-%{version}.tar.xz.asc
+Source2:       https://people.redhat.com/~ssorce/simo_redhat.asc
```

Comment:
If I remember correctly, Fedora's packaging guidelines do not allow conditional sources, because you're supposed to be able to specify the option when building from an existing srpm, but that would fail because the srpm would not contain the source files if it was built without the `gpgcheck` option.

Unfortunately I don't see a better way to handle this, either, since the `.asc` file would 404 for development builds and thus fail the srpm build.

Reply:
I found only a note about arch-specific conditions:

https://docs.fedoraproject.org/en-US/packaging-guidelines/#_no_arch_specific_sources_or_patches

Reply:
It's here: https://docs.fedoraproject.org/en-US/packaging-guidelines/SourceURL/#_do_not_conditionalize_sources, introduced in https://pagure.io/packaging-committee/pull-request/1163.

Reply:
%bcond_with gpgcheck should take care of it